.github/workflows/release_cli_and_assets.yml

```diff
@@ -324,8 +324,8 @@ jobs:
           - "blank_aggregators"
           - "search_path"
           - "chaos_and_query"
-          - "dynamic_schema"
-          - "dynamic_aggregators"
+          # - "dynamic_schema"
```

Comment:
rather than comment them out, please skip the tests with a reason and a link to the issue